src/components/BatchVoting/VotingOptions/TinderCardComponent.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { FC, useState } from 'react';
+import { Button, Divider } from 'antd';
+import Markdown from '~src/ui-components/Markdown';
+import { IVotesCount } from '~src/types';
+import Link from 'next/link';
+import { ITinderCardsComponent } from '~src/components/TinderStyleVoting/types';
+import CardPostHeading from '~src/components/TinderStyleVoting/PostInfoComponents/CardPostHeading';
+import CardComments from '~src/components/TinderStyleVoting/CardComponents/CardComments';
+import ReferendumV2CardInfo from '~src/components/TinderStyleVoting/PostInfoComponents/ReferendumV2CardInfo';
+
+const TinderCardsComponent: FC<ITinderCardsComponent> = (props) => {
+	const { proposal, onSkip } = props;
+	const [ayeNayAbstainCounts, setAyeNayAbstainCounts] = useState<IVotesCount>({ abstain: 0, ayes: 0, nays: 0 });
+
+	const sanitizeSummary = (md: string) => {
+		const newMd = (md || '').trim();
+		return newMd;
+	};
+
+	return (
+		<div className='flex overflow-x-hidden'>
+			<div className='overflow-y-auto rounded-2xl bg-white px-4 py-6 dark:border dark:border-solid dark:border-separatorDark dark:bg-black'>
+				<div className='flex items-start justify-between'>
+					<CardPostHeading
+						method={proposal?.method}
+						motion_method={proposal?.motion_method}
+						postArguments={proposal?.proposed_call?.args}
+						className='mb-5'
+						post={proposal}
+					/>
+					<Button
+						className='border-none bg-transparent p-0 text-pink_primary'
+						onClick={() => onSkip(proposal.id)}
+					>
+						Skip &gt;
+					</Button>
+				</div>
+				<Divider
+					type='horizontal'
+					className='border-l-1 -mt-4 mb-4 border-[#D2D8E0] dark:border-separatorDark'
```

Comment:
hard coded border colour here, not good for dark mode etc

Reply:
resolved